src/routes/project/TaskRow.svelte

```diff
@@ -144,4 +175,19 @@
 		border-radius: 50%;
 		overflow: hidden;
 	}
+	.sticky-name {
```

Comment:
Are we BEM or not BEM? This should be modifier, I guess. And move it, please, closer to those things it modifies.